buildout.cfg

```diff
@@ -28,7 +28,7 @@ input = inline:
    python bootstrap-buildout.py --setuptools-version=8.3
    ${buildout:directory}/test.plone_addon/bin/buildout
    ${buildout:directory}/test.plone_addon/bin/test
-   rm -rf ${buildout:directory}/test.plone_addon/
+   # rm -rf ${buildout:directory}/test.plone_addon/
```

Comment:
Yeah. Kind of. For debugging I kept the generated code. Should not be committed though.